common/variant/README.md

```diff
@@ -362,6 +362,8 @@ The Decimal type contains a scale, but no precision. The implied precision of a
 | 18 <= precision <= 38 | int128             |
 | > 38                  | Not supported      |
 
+The *Logical Type* column indicates logical equivalence of physically encoded types. For example, a user expression operating on a string value containing "hello" should behave the same, whether it is encoded with the short string optimization, or long string encoding. Similarly, user expressions operating on an *int8* value of 1 should behave the same as a decimal16 with scale 2 and unscaled value 100.
```

Comment:
If `int8` and other int types are `Exact Numeric` and decimals are `Exact Numeric` how do they relate when the decimal values can represent non-integers? For example, a decimal with scale 2 and unscaled value 123 is an `Exact Numeric` and an `int8` with value 1 is also a `Exact Numeric`. Those seem like different classes of values?

Reply:
I'm not sure I understand the question, but they're both numbers with a well-defined value. If a number can be represented with either physical representation (e.g. 1), then the value should be considered the same, whether it's encoded as an `int8` or `decimal(16, 2)`. If It can only be represented by one of them (e.g. 1.23), then there's no way to encoded it using `int8`. If an engine wants to support e.g. comparison of variant values, than `1.23 > 1` should be true regardless of the underlying representation of the two values.

Reply:
Ahhh, I see, it makes sense to me now. They are all "exact numerics", but depending on the actual value, it may end up in a different physical type, like decimal, or int.